nni/runtime/protocol.py

```diff
@@ -27,6 +27,7 @@ class CommandType(Enum):
     SendTrialJobParameter = b'SP'
     NoMoreTrialJobs = b'NO'
     KillTrialJob = b'KI'
+    ReportSearchSpace = b'RS'
```

Comment:
I thought the visual search space could be directly embedded into original search space, rather than requiring a new channel.

Reply:
Because we should add a new transmission direction, i.e., sending search space from tuner/advisor to nnimanager. Previously, search space is sent from nnimanager to tuner/advisor

Reply:
We already have update search space in REST API. See `Experiment.update_search_space()` for an example implementation.
I think Retiarii should report search space use this method.

Reply:
update search space is from nnimanager to advisor, not advisor to nnimanager

Reply:
`Experiment.update_search_space()` is from experiment launcher (`RetiariiExperiment`) to NNI manager, and then to dispatcher.

Reply:
updated to using this function